en/micro-integrator/docs/administer-and-observe/using-the-analytics-dashboard.md

```diff
@@ -65,6 +65,12 @@ Let's start the servers in the given order.
 !!! Warning
     Be sure to [configure the SSL certificates](../../setup/observability/setting-up-classic-observability-deployment/#configure-ssl-certificates) before starting the Micro Integrator.
 
+#### Configuring the Micro Integrator to Publish Data to Analytics Server
+Analytics publishing can be configured in `<EI_ANALYTICS_HOME>/conf/carbon.xml` in 'Analytics' section.
```

Comment:
this needs to be MI_HOME ? and we need to give instructions on how to configure this in toml ?